aks/application/variables.tf

```diff
@@ -92,8 +92,20 @@ variable "probe_command" {
   description = "Command for the liveness and startup probe"
 }
 
-locals {
-  http_probe_enabled = var.is_web && var.probe_path != null
-  exec_probe_enabled = !var.is_web && length(var.probe_command) != 0
-  probe_enabled      = local.http_probe_enabled || local.exec_probe_enabled
+variable "enable_statuscake" {
+  type        = bool
+  default     = true
+  description = "Whether to set up StatusCake alerts for web applications"
+}
+
+variable "statuscake_contact_groups" {
+  type        = list(string)
+  default     = []
+  description = "Contact groups for the StatusCake alerts"
+}
+
+variable "statuscake_confirmations" {
```

Comment:
Can we keep it hardcoded to 2? We always use 2 I think.
If there is a requirement to change it later on, we can introduce a new variable. For now, it can be as a "constant" in a local var.

Reply:
Ah nice, I wasn't sure if it changed, for some reason on Apply for QTS we'd made it a variable but always set it to 2, so I just copied that.

Reply:
I've amended the commit to hard code this value.